client/src/components/Card/Card.jsx

```diff
@@ -0,0 +1,25 @@
+import "./Card.css";
+
+function Card({ image, altText, date, tag, title, summary }) {
```

Comment:
We will likely later need the id as a prop to get that specific Page (on button click) so we might as well put it in here now

```suggestion
function Card({ id, image, alternativeText, date, tag, title, summary }) {
```